promotion/src/main/java/com/yas/promotion/model/Promotion.java

```diff
@@ -24,11 +24,30 @@ public class Promotion extends AbstractAuditEntity{
 
     private String couponCode;
 
+    private Long value;
+
+    private Long amount;
+
+    private boolean isActive=false;
```

Comment:
I don't understand the purpose of setting the default value for isActive as false.